### DISCUSSION TITLE
Why are all methods of getting time ".utc() "?

### DISCUSSION THREAD

**Discussion by Blueda** [requarks/wiki#6176] [Help / Questions] [answered]
Why are all methods of getting time ".utc() "?
Why not “.local()”?
For example:
"server/models/pageHistory. js" line 240
The code is as follows:
`const olderThanISO = DateTime.utc().minus(dur)`
"server/models/apiKeys.js" line 34
The code is as follows:
`this.updatedAt = moment.utc().toISOString()`
My server time is CTS (UTC+08).
The time saved to the database is incorrect. How can I solve the problem?

**Comment by NGPixel** [comment, marked as answer]
Because it doesn't make sense to store datetime in a local timezone.
By using UTC, it makes the data portable, no matter where it is stored.

The date time is only converted to the local timezone on the client side, at render time.

**Reply by Blueda** [reply]
However, the date displayed on the client is wrong, and there is a difference of 13 hours with my local time. How to ask the client to display the UTC time string stored in the database as the time in my current time zone?

**Reply by NGPixel** [reply]
Did you set your timezone in your profile?

**Reply by Blueda** [reply]
The server time is:
      Local time: 二 2023-02-21 14:29:27 CST
  Universal time: 二 2023-02-21 06:29:27 UTC
        RTC time: 二 2023-02-21 06:29:27
       Time zone: Asia/Shanghai (CST, +0800)
     NTP enabled: yes
NTP synchronized: yes
 RTC in local TZ: no
      DST active: n/a
----------------------------------------------
My computer time is:
2023年 2月21日 星期二 14时29分30秒 CST
----------------------------------------------
I just updated a document
The Wiki.js document shows the time as:
Today at 1:29 AM
